cloudbuild/presubmit.sh

```diff
@@ -38,7 +38,7 @@ initialize_git_repo() {
 }
 
 # Determines whether a given string is a prefix string of any changed file name
-is_prefix() {
+is_changed_prefix() {
   for file in "${CHANGED_FILES[@]}"; do
     if [[ $file =~ ^$1 ]]; then
```

Comment:
This condition is hard to understand, add comment. Also assign $1 to a meaningful local variable.

Reply:
Deleted this function.